HelpSource/Overviews/Documents.html

```diff
@@ -5,15 +5,16 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta http-equiv='Content-Type' content='text/html; charset=UTF-8' />
     <link rel='stylesheet' href='../scdoc.css' type='text/css' />
+    <link rel='stylesheet' id='scdoc-theme' href='../themes/default.css' type='text/css' />
     <link rel='stylesheet' href='../frontend.css' type='text/css' />
     <link rel='stylesheet' href='../custom.css' type='text/css' />
     <script src="../lib/jquery.min.js"></script>
     <script>
         var scdoc_title = "Documents";
         var scdoc_sc_version = "";
     </script>
-    <script src="../docmap.js" type="text/javascript"></script>
     <script src="../scdoc.js" type="text/javascript"></script>
+    <script src="../docmap.js" type="text/javascript"></script>
```

Comment:
Why is it necessary to load these files in this order now? Should they be dependent on each other?